stack.yaml

```diff
@@ -55,8 +55,8 @@ packages:
   extra-dep: true
 
 - location:
-    git: https://github.com/input-output-hk/cardano-crypto
-    commit: 33c7ecc6e4bd71c3ea0195e9d796eeace7be22cf
+    git: https://github.com/anviking/cardano-crypto # FIXME: Don't use fork
+    commit: 83d320432d48cb643c923ce7d15da5ba119a24c6
```

Comment:
We can pick up the right revision from cardano-crypto now that it has been merged (e87fcd3fea18f664d6e4f39c11abb8cafbd25785).

Note that this can safely be done since these changes are the only one introduced recently on cardano-crypto:

https://github.com/input-output-hk/cardano-crypto/compare/33c7ecc6e4bd71c3ea0195e9d796eeace7be22cf...e87fcd3fea18f664d6e4f39c11abb8cafbd25785